hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/AbstractMemStore.java

```diff
@@ -95,7 +96,10 @@ protected void resetActive() {
         memstoreAccounting.getHeapSize(), memstoreAccounting.getOffHeapSize(),
         memstoreAccounting.getCellsCount());
     }
-    timeOfOldestEdit = Long.MAX_VALUE;
+  }
+
+  protected void resetTimeOfOldestEdit() {
+    this.timeOfOldestEdit = Long.MAX_VALUE;
```

Comment:
Is Long.MAX_VALUE the initialized value? It gets changed when the next edit comes in?

Reply:
yes, is Long.MAX_VALUE. it get change when  flushInMemory.